js/mcqModel.js

```diff
@@ -168,10 +168,59 @@ define([
                 _selectedItems: [],
                 _isAtLeastOneCorrectSelection: false
             });
+        },
+
+        getInteractionObject: function() {
+            var interactions = {
+                correctResponsesPattern: [],
+                choices: []
+            };
+
+            interactions.choices = this.get('_items').map(function(item) {
+                return {
+                    id: (item._index + 1).toString(),
+                    description: item.text
```

Comment:
that's global to the course, i'm not sure it's necessary to include that with every call?